cmd/dep/testdata/harness_tests/README.md

```diff
@@ -51,7 +54,7 @@ The test procedure is as follows:
 2. Create `src/github.com/golang/notexist` as the project
 3. Copy the contents of `initial` to the project
 4. `go get` the repos and versions in `imports` to the `src` directory
```

Comment:
Let's say `fetch` rather than `go get`; less misleading.

Also, should probably note in here that we only support git repos.